### DISCUSSION TITLE
Custom Manifests

### DISCUSSION THREAD

**Discussion by vsoch** [supercontainers/containers-wg#10] [Ideas]
This discussion is meant to track this idea: 

https://supercontainers.github.io/containers-wg/ideas/dynamic-manifests/

**Comment by sk8forether** [comment]
I can imagine this being an expansion upon modulefiles ([Lmod](https://lmod.readthedocs.io/en/latest/)).  So much like a user loads and unloads modules, one could imagine users dynamically building a container by adding and removing certain software.

**Reply by vsoch** [reply]
It's not a dynamic build, but if you are interested in Lmod + containers check out Singularity Registry HPC! (note conflict of interest, I am the developer of said library :laughing: )

- https://singularity-hpc.readthedocs.io/en/latest/
- https://www.youtube.com/watch?v=xvi-Om8jj2E

But I do think the "custom build" is something we should strive for. As we said in the meeting: clickclickclick and go!
